File: config/config-sbc2ha/src/main/java/com/dfi/sbc2ha/config/sbc2ha/definition/output/GpioOutputConfig.java
package com.dfi.sbc2ha.config.sbc2ha.definition.output;

import com.dfi.sbc2ha.config.sbc2ha.definition.enums.OutputKindType;
import lombok.Data;
import lombok.EqualsAndHashCode;

@EqualsAndHashCode(callSuper = true)
@Data
public class GpioOutputConfig extends OutputConfig {


    public GpioOutputConfig() {
        super();
        kind = OutputKindType.GPIO;
    }
}

File: config/editor/src/app/definition/actuator/GpioOutputConfig.ts
import {ActuatorConfig} from "./ActuatorConfig";
import {OutputKindType} from "../enums/OutputKindType";
import {ActuatorType} from "../enums/ActuatorType";


export class GpioOutputConfig extends ActuatorConfig {

  constructor(name: string, outputType: ActuatorType, output: number) {
    super(OutputKindType.GPIO, name, outputType, output);
  }
}
